services/libreTranslate.js

```diff
@@ -1,11 +1,16 @@
 import fetch from "node-fetch";
+import { getConfig } from "../utils/getConfig.js";
 
-const mirrors = [
+const defaultMirrors = [
   "https://translate.terraprint.co/translate",
   "https://trans.zillyhuhn.com/translate",
 ];
 
 const libreTranslate = async (text, { from, to }) => {
+  const config = await getConfig();
+  const customMirrors = config?.mirrors || [];
+  const mirrors = [...customMirrors, ...defaultMirrors];
```

Comment:
Instead of importing the config directly here, we should import it on the translateController and then pass it down until here.

https://github.com/heliomar-pena/i18n-populator/blob/ca74d0b21a89cf156a9731251d3b8d35f0ce781c/controllers/translateController.js#L40-L44